apps/web/.env.example

```
@@ -1,7 +1,11 @@
 NEXT_PUBLIC_SANITY_PROJECT_ID=
-NEXT_PUBLIC_SANITY_DATASET=
+NEXT_PUBLIC_SANITY_DATASET=production
 NEXT_PUBLIC_SANITY_API_VERSION=2025-08-29
 NEXT_PUBLIC_SANITY_STUDIO_URL=http://localhost:3333
 
 SANITY_API_READ_TOKEN=
 SANITY_API_WRITE_TOKEN=
+
+SHOPIFY_STORE_DOMAIN=
+SHOPIFY_STOREFRONT_ACCESS_TOKEN=
+SHOPIFY_API_VERSION=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)